files/image_config/hostcfgd/hostcfgd.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Host config enforcer daemon
+Requires=database.service
+After=database.service
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/hostcfgd 
```

Comment:
where is this hostcfgd, what is the usage of this hostcfgd? I cannot find this file in the PR. I suggest to have a separate PR for hostcfgd.

Reply:
hostcfgd has been created as another PR in sonic-utilities  https://github.com/Azure/sonic-utilities/pull/125